src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -1,35 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Scroll from 'react-scroll';
 
 import InfoPair from './InfoPair';
 
+import { getScrollOptions } from '../../common/utils/helpers';
 import { formatDateShort } from '../utils/helpers';
 
+const scroller = Scroll.scroller;
+
 class EnrollmentPeriod extends React.Component {
+  constructor() {
+    super();
+    this.scrollToEnrollment = this.scrollToEnrollment.bind(this);
+    this.toggleHistory = this.toggleHistory.bind(this);
+    this.state = { historyExpanded: false };
+  }
+
+  scrollToEnrollment() {
+    const options = getScrollOptions({ delay: 2 });
+    scroller.scrollTo(`collapsible-${this.props.id}`, options);
+  }
+
+  toggleHistory() {
+    const shouldExpand = !this.state.historyExpanded;
+    this.setState({ historyExpanded: !this.state.historyExpanded });
+    if (shouldExpand) {
+      this.scrollToEnrollment();
+    }
+  }
+
   render() {
-    const { enrollment } = this.props;
+    const { enrollment, id } = this.props;
+    const amendments = enrollment.amendments || [];
+
+    // TODO: should fullTimeHours be used anywhere?
+    // const fullTimeHours = <InfoPair label="Full-time Hours" value={enrollment.fullTimeHours}/>;
+    const onCampusHours = <InfoPair label="On-campus Hours" value={enrollment.onCampusHours}/>;
+    const onlineHours = <InfoPair label="Online Hours" value={enrollment.onlineHours}/>;
 
-    const fullTimeHours = <InfoPair label="On-campus Hours" value={enrollment.fullTimeHours}/>;
-    const distanceHours = <InfoPair label="Distance Hours" value={enrollment.distanceHours}/>;
-    const typeOfChange = enrollment.amendmentList && enrollment.amendmentList.length > 0 &&
-      <InfoPair label="Type of Change" value={enrollment.amendmentList[0].type}/>;
-    const changeEffectiveDate = enrollment.amendmentList && enrollment.amendmentList.length > 0 &&
-      <InfoPair label="Change Effective Date" value={enrollment.amendmentList[0].changeEffectiveDate}/>;
+    const amendmentList = this.state.historyExpanded ? (
+      <div id={`collapsible-${id}`} className="usa-accordion-content">
+        {amendments.map((amendment, index) => {
+          return (
+            <div key={`amendment-${this.props.key}-${index}`}>
+              {index > 0 ? <hr/> : null}
```

Comment:
I styled `hr` to have no margin on the bottom for this page, because it was creating styling issues elsewhere. But with the amendment list some margin bottom might be needed because it looks a little funny in the screenshot. What I might recommend is reversing the change on `hr` specifically to not have `margin-bottom: 0;`, and instead collapse the top margin of the next sibling element (specifically, we want to do this where we have a `hr` followed by a `h4`. So maybe something like this:

```
hr {
  border-style: solid;
  border-color: $color-gray-light;
  border-width: 1px 0 0 0;
  
  & + h4 {
    margin-top: 0;
  }
}
```

I think something like this will work, but not able to test right now because the data isn't populating when I run this locally. Let me know if this doesn't make sense!